### PR TITLE
Suggesting few changes to the doc

### PR DESCRIPTION
**Pull request by Mohit-Nain**
**What this PR does / why we need it**:
1. Note should be followed by a dot (.)
2. Correcting a typo in default
3.  Re-writing sub sentence to make reading better

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/installation/configuration.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Another way is put a webserver like Nginx or Apache in front of Grafana and have
 
 `http`,`https`,`h2` or `socket`
 
-> **Note** Grafana versions earlier than 3.0 are vulnerable to [POODLE](https://en.wikipedia.org/wiki/POODLE). So we strongly recommend to upgrade to 3.x or use a reverse proxy for ssl termination.
+> **Note.** Grafana versions earlier than 3.0 are vulnerable to [POODLE](https://en.wikipedia.org/wiki/POODLE). So we strongly recommend to upgrade to 3.x or use a reverse proxy for ssl termination.
```
Comment (aknuds1) [inline docs/sources/installation/configuration.md]: What's the reasoning behind adding the period? We don't do this anywhere else in our docs either.

File: docs/sources/installation/configuration.md [hunk 2]
```diff
@@ -415,7 +415,7 @@ organization to be created for that new user.
 
 Set this value to automatically add new users to the provided org.
 This requires `auto_assign_org` to be set to `true`. Please make sure
-that this organization does already exists.
+that this organization should be pre-existing.
```
Comment (aknuds1) [inline docs/sources/installation/configuration.md]:
```suggestion
that this organization already exists.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: docs/sources/installation/configuration.md (hunk 1 above)
Comment (Mohit-Nain) [reply docs/sources/installation/configuration.md]: it is there in the previous Note section, just makes the look and feel of the document little better

Comment (aknuds1) [reply docs/sources/installation/configuration.md]: Oh yeah, I see there are few now (made a mistake when searching earlier). Personally I don't see the point of adding the period. What do you think @oddlittlebird?

Comment (oddlittlebird) [reply docs/sources/installation/configuration.md]: Instead of a period, use a colon `:` 

**Note:**

Comment (Mohit-Nain) [reply docs/sources/installation/configuration.md]: Used colon , instead of period. Please merge the changes.

**Review by aknuds1 — APPROVED** [review]
LGTM! Thanks for contributing to Grafana!

**Pull request merged**